test/tests/composer.js

```diff
@@ -1709,6 +1709,50 @@ module.exports = Nodal => {
 
     });
 
+    it('Should throw errors when querying with a bad column name', done => {
+
+      try {
+
+        Parent.query()
+          .join('pets')
+          .where({ active: true })
+          .end();
```

Comment:
Keep the style consistent with other tests (use the callback and catch the error)

Reply:
The way I wrote the code is such that it throws an error outside of the `end()` call. I did this is because it's more of a developer error, not a DB one. If I did it inside the end call then it might be less obvious.